WorkflowUI/Sources/Screen/AdaptedEnvironmentScreen.swift

```diff
@@ -30,7 +30,7 @@ import ViewEnvironment
 ///     .adaptedEnvironment(keyPath: \.myValue, to: newValue)
 /// ```
 ///
-public struct AdaptedEnvironmentScreen<Content: Screen>: Screen {
```

Comment:
I'll defer to @bencochran here since he's the one who usually structures things this way, but eg, it's common to build a back stack wrapping some type of view model, etc, and then map that into a screen type after you fetch it. This allows you to use environment adaptation alongside that.